### DISCUSSION TITLE
AI生成簡報工具:Gamma

### DISCUSSION THREAD

**Discussion by SusuTima** [1101-GS4719-C/Python-Community-GS4719-C#1920] [General]
相信大家在不管是通識課、必修課都會遇到要做簡報的時刻。
這邊想跟大家分享一個好用的AI工具:Gamma，他是一個可以自動生成簡報的AI工具，在進入Gamma創立帳號後，選擇想要的簡報張數，接著在搜尋框輸入關鍵字，最後選擇背景主題，按下ENTER，一份與主題有關的簡報就被生成出來了。簡報生成後，右邊有一排紫色的，最上面的按鍵是與AI溝通，讓他依照你的指示調整簡報(例如:希望簡報更簡潔一點)，下面的鍵則是可以匯入照片、影片、表格。以「AI的歷史沿革」為主題，以下為Gamma生成的簡報:
![簡報生成1](https://github.com/NCU-GS4719-Python/Python-Community/assets/132805404/a361d8c4-efd5-4fd5-b158-670b463759f9)
![簡報生成2](https://github.com/NCU-GS4719-Python/Python-Community/assets/132805404/25502e26-e341-40d3-892b-1f666dbb9464)
![簡報生成3](https://github.com/NCU-GS4719-Python/Python-Community/assets/132805404/bd9771ee-eff4-4922-8e36-e6ada54f5e77)
![簡報生成4](https://github.com/NCU-GS4719-Python/Python-Community/assets/132805404/3ceea63f-f481-4fcc-8a46-9b8fffd184e8)
![簡報生成5](https://github.com/NCU-GS4719-Python/Python-Community/assets/132805404/c54f688e-9bdd-4997-80e4-07d81d56d304)
![簡報生成6](https://github.com/NCU-GS4719-Python/Python-Community/assets/132805404/36a3b0b6-ea02-40fd-8c4d-6913e15a42be)
![簡報生成7](https://github.com/NCU-GS4719-Python/Python-Community/assets/132805404/d37e741d-673f-4f70-9175-9e210c645c8b)

有了這個工具，可以以AI生成的簡報為基礎，再加上自己想加上的內容，也可以省下大量排版的時間，是我們做簡報時的好幫手。

**Comment by Eileen050824** [comment]
我覺得這個工具的確非常方便，特別是非常符合我們大學生的需求，由AI幫忙擬定大部分的瑣事，同時又能按照我們的意思生出大綱，是相當實用的作業幫手。但如果單看上面的AI的成果，我自己會覺得一切太過模板工整化，可能會少了很多個人色彩和趣味性，這也再強調了這是個很棒的AI工具，但我們仍需靠自己努力完成更豐富的作品，而非單純丟給AI生成！謝謝同學推薦這個好物，有機會我會試用看看👍

**Comment by YifenLin** [comment]
對於簡報製作，我認為使用Gamma這樣的AI工具確實能夠節省許多時間與精力。然而，相較於PowerPoint或Google Slides，雖然Gamma提供了快速生成簡報的功能，但在自由度與客製化方面可能稍有不足。對於需要特定風格或深度內容的簡報而言，仍然需要人工的干預與調整。因此，在選擇工具時，應根據需求和情境進行選擇，以確保最終呈現的效果符合期望。

**Comment by KH1126** [comment]
我認為Gamma這樣的AI工具對於做簡報的人來說是一個非常有用的工具。首先，它能夠自動生成簡報，節省了大量的時間和精力，尤其是對於需要經常做簡報的人來說，這樣的工具能夠大大提高工作效率。

其次，Gamma可以根據用戶的需求和主題生成相關的簡報，這樣用戶不需要自己花費時間去搜尋和整理資料，Gamma已經為他們完成了這些工作。這個AI工具使得簡報製作變得更加輕鬆和高效。

**Comment by wanderer657** [comment]
用Gamma做簡報真是太方便了！只要幾個步驟就能快速生成簡報，省去了許多排版時間。我最喜歡的是可以跟AI溝通，根據自己的需求調整簡報風格，真是太貼心了。而且可以輕鬆地加入照片、影片等內容，讓簡報更生動有趣。用了Gamma後，我發現自己在製作簡報時節省了不少時間，也更專注在內容上。這工具真是太棒了，以後做簡報就靠它了！

**Comment by benbebobo149** [comment]
起初，我和你一樣，對Gamma感到非常期待，認為它會成為我們學生的救星。然而，隨著我使用的次數增多，我發現它的功能並不像我最初想像的那麼強大。它似乎還處於偽AI的階段，並沒有真正具備智能。每次使用Gamma時，它似乎只是根據類似的模板來生成內容，而缺乏創意和靈活性。這讓我開始思考，是否還有其他更有效的方式來完成我想要做的事情。最後，我要感謝同學分享這個觀點，讓我們能夠更深入地了解Gamma的局限性。

**Comment by lyykey** [comment]
謝謝同學的分享，起初我對 Gamma 抱著相當的期待，認為以後報告人生就靠它了。但我後來覺得 AI 模型就目前而言，終究是模型，如果對於同一個對象報告，很容易會被發現模板幾乎一模一樣，沒有特別的改動，因此我認為這是 Gamma 的缺點，雖然很方便，對於框架能有很快的實現，不過要做到創新、獨一無二，還是需要技術的創新，最後再次謝謝同學的分享~

**Comment by tzurou** [comment]
我也是前陣子用ppt做簡報才意外發現Gamma這個AI生成式簡報工具，但還沒用過，看板主生出來的簡報感覺很精緻，但就是缺乏一點人類的溫度，不過整體而言很不錯，謝謝版主的分享，讓我找到另一個簡報製作工具。

**Comment by 710388** [comment]
看起來很不錯欸，下次通識報告我搞不好會試試看，謝謝分享。有人試過把gpt插到PowerPoint 裡面嗎？

**Comment by cassy0201** [comment]
今天第一次了解這個工具，感覺很方變又有效率，下次會想試試看!

**Comment by NcuJerryKao** [comment]
這個分享的Gamma AI工具確實聽起來非常方便和實用，能夠幫助用戶節省時間和精力在製作簡報上。以下是一些建議回覆：
1. 體驗和實際應用：建議您在使用Gamma AI工具後，深入體驗其功能和效能
2

**Comment by YeeDaniel** [comment]
Gamma利用人工智慧技術自動生成簡報內容，根據用戶提供的主題和要點創建合適的文本、圖表和圖片。
提供多種專業設計的模板，用戶可以根據需求選擇，確保簡報外觀美觀且一致。

**Comment by sunset513** [comment]
我這裡想提供其他建議，對簡報製作也很有幫助。

我自己會先將內容請 gpt 做初步整理，整理完後再請他將內容改成簡報的架構。得到了簡報架構後再透過 gamma 生成簡報，簡報的品質可能會更佳一點!我這麼做是因為 gpt 比較方便透過對話修改，這能夠提升一點效率。

另外 gamma 最近也增加了生成圖片的功能，超方便的。我不用再花太多時間去找搭配簡報的圖片了，我的效率提升了不少!

**Comment by hunteryang0108** [comment]
使用Gamma輔助製作簡報確實是個不錯的方法，自動配圖的功能也實在令人驚艷，謝謝同學的分享！但是，以這樣的工具製作出的簡報是否就沒有了原創性？雖然用在介紹一些簡單概念是沒有問題的，但若是要用在專題或研究結果的展示可能就不是那麼適合。

**Comment by Nana1218** [comment]
我覺得Gamma是一個很好的工具，對於做簡報而言，但我認為他比較適合當輔助，不適合做為主要工具，因為有一些東西是需要加入更多自己的想法的，如果用Gamma作感覺會限縮了創意。

**Comment by kingsophy** [comment]
第一次聽到這個工具是在高中的一堂講座上，後來自己用過之後發現真的很好用，不管是在做網站還是簡報都幫助很大，不過仍然需要自己修改很大一部份，只是給一個大概，但還是很好用。

**Comment by ds0w0** [comment]
感謝你分享 Gamma 這個好用的 AI 工具！它的確可以在短時間內生成簡報，省去大量排版的時間，對於急需準備簡報的人來說非常方便。我以前在體育課的報告時有用過，雖然它能夠生成簡易明瞭的簡報，不過，值得注意的是，AI 生成的簡報有時顯得過於公式化，缺乏個人特色，而且無法藉由平實的畫面吸睛。由於它依賴模板和預設內容，無法完全展現你的獨特觀點和個人風格。此外，缺少手動製作簡報的過程，可能會讓你在報告時對內容不夠熟悉，影響展示效果和對問題的深入理解。雖然 AI 工具可以大大提高效率，但在準備簡報時，還是建議適度地加入自己的思考和創意，這樣不僅能夠展現個人風格，也能加深對內容的掌握。

**Comment by poyen0806** [comment]
感謝你分享 Gamma 這個好用的 AI 工具！以下是我的使用心得

一開始用的時候覺得很新奇
但後來逐漸覺得還是自己做會比較好
畢竟自己做會更清楚簡報內容
而不是先用講稿生成後
還要在一個一個對

所以如果要我選的話
我還是會比較常用canva跟powerpoint
而不會常用gamma
喔對了gamma還要點數...

**Comment by XiangLuen** [comment]
如果是我的話，我覺得這種 AI 生成簡報的工具確實很方便，特別是臨時被丟一個報告，或是沒什麼頭緒時，至少可以先弄個大概的架構出來。不過，這種東西生成出來的內容通常比較中規中矩，很多時候還是得自己改，不然就會有一種「這報告不是你做的吧？」的感覺。
而且我認為簡報，最關鍵的其實不是排版多漂亮，而是內容有沒有邏輯，還有講的人能不能抓住聽眾的注意力。AI 可以幫你省時間，但如果只是套用它丟出來的東西，自己連內容都沒完全消化，那上去講的時候還是會很空洞。所以，這類工具能用，但不能太依賴，畢竟真正的簡報能力，還是得靠自己練起來。

**Comment by SendaiHatsuki** [comment]
AI 生成 PPT 確實很方便，可以快速產出簡報，但說實話，品質還是有點差。以我的經驗來看，用 AI 做出來的 PPT 會有一種很明顯的「機械感」，排版和配色雖然很漂亮有時候看起來不太協調，內容呈現也比較死板。當然，如果只是臨時需要一份簡報，AI 還是蠻好用的，但如果想要做出可以當作業繳交、個性化的 PPT，還是要自己做才行。

**Comment by lichilun** [comment]
我覺得Gamma這個工具很方便，適合在時間有限或需要快速製作基礎內容的情況下使用，不過，我覺得還是有一些優缺點。

優點：
1.節省時間：AI自動生成簡報框架，減少手動編排的時間。
2.與AI互動調整：透過與AI溝通，調整簡報的簡潔度或語氣，這讓工具不只是靜態生成，還具備靈活性。
3.多媒體整合：自動匯入照片、影片、表格，增加簡報的豐富度與視覺吸引力。

缺點：
1.內容深度：AI生成的簡報可能會比較淺層，無法涵蓋專業或細節豐富的內容。
2.主題相關性：生成的內容可能無法，真正貼合特定主題。
3.美感與設計：背景主題雖然自動生成，但美感可能變得單一化。
（這些缺點，需要編輯者需要進一步補充調整、編輯！）
